roles/kubernetes/control_plane/init/primary/templates/kubeadm-config.j2

```diff
@@ -37,7 +44,11 @@ clusterName: "{{ kubernetes_cluster_name }}"
 #
 # Kubelet Configuration : https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/
 #
+{% if kubernetes_version is version('1.31.0', 'lt', version_type='semver') %}
 apiVersion: kubelet.config.k8s.io/v1beta1
+{% else %}
+apiVersion: kubelet.config.k8s.io/v1beta2
+{% endif %}
```

Comment:
I don't see v1beta2 for this config. Only v1beta1 and v1alpha1 along with v1. This needs to change. Link - https://kubernetes.io/docs/reference/config-api/kubelet-config.v1/